Adding release-note for 0.14.1 (#863)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

weight = 40
+++
<!-- markdownlint-disable no-duplicate-header -->
## v0.14.1

This is a bugfix release:

* Stop using cluster-owned tag for AWS Security Group lookup.

> * Run `subctl diagnose firewall` command with individual kubeconfigs will now override the default namespace to `submariner-operator`.

Reviewer: ```suggestion
* Running the `subctl diagnose firewall` command with individual kubeconfigs will now override the default namespace to `submariner-operator`.
```

> * Running the `subctl diagnose firewall` command with individual kubeconfigs will now override the default namespace to `submariner-operator`.

Reviewer 2: ```suggestion
* Running the `subctl diagnose firewall` command with individual kubeconfigs will now deploy diagnose pods in the  `submariner-operator` namespace to avoid podSecurity errors.
```

> * Fix issues with gateway public IPs resolver.

Reviewer 2: ```suggestion
* The periodic public-ip watcher is enhanced to use random external servers to resolve the public IP associated with Gateway nodes.
```

> * The `subctl diagnose kube-proxy-mode` now also checks if IPVS is enabled or not.

Reviewer: ```suggestion
* The `subctl diagnose kube-proxy-mode` command now also checks if IPVS is enabled or not.
```

> * The `subctl diagnose kube-proxy-mode` command now also checks if IPVS is enabled or not.

Reviewer 2: The subctl diagnose kube-proxy-mode already supported the IPVS validation. The issue was seen with downstream container images where the iproute package returns a different error message.

```suggestion
* The `subctl diagnose kube-proxy-mode` command now works fine with different versions of iproute packages.
```

weight = 40
+++
<!-- markdownlint-disable no-duplicate-header -->
## v0.14.1

This is a bugfix release:

* Stop using cluster-owned tag for AWS Security Group lookup.

> * Running the `subctl diagnose firewall` command with individual kubeconfigs will now deploy diagnose pods in the  `submariner-operator` namespace

Reviewer 3: ```suggestion
* Running the `subctl diagnose firewall` command with individual kubeconfigs will now deploy diagnose pods in the `submariner-operator` namespace
```

> to avoid podSecurity errors.

Reviewer 3: ```suggestion
to avoid pod security errors.
```

If we really mean to talk about an object, I think it would be `PodSecurity`.

* The periodic public IP watcher is enhanced to use random external servers to resolve the public IP associated with Gateway nodes.

> * The `subctl diagnose kube-proxy-mode` command now works fine with different versions of iproute packages.

Reviewer 3: ```suggestion
* The `subctl diagnose kube-proxy-mode` command now works with different versions of iproute packages.
```

* The following changes were made to pods running `subctl diagnose` commands in order to allow them to run commands like `tcpdump`:

>   * Make the `diagnose` pod a privileged.

Reviewer 3: ```suggestion
  * Make the `diagnose` pod privileged.
```

>   * Run the `diagnose` pod with userId 0.

Reviewer 3: ```suggestion
  * Run the `diagnose` pod with user ID 0.
```
